### PR TITLE
Cindy start

### PR DESCRIPTION
**Pull request by cindyciclta**

### CODE REVIEW

File: src/chirp_client.cc
Comment (barath) [inline src/chirp_client.cc:110]: Minor: I'd recommend having reformatting be a separate pull request.  But it's fine.

File: src/chirp_client.cc
Comment (barath) [inline src/chirp_client.cc]: Maybe use log statements instead?

File: src/chirp_client.cc
Comment (barath) [inline src/chirp_client.cc:154]: What does 1 mean?

File: src/chirp_client.cc
Comment (barath) [inline src/chirp_client.cc]: Logging?

**Comment by barath** [comment]
You should also remove all the non-code files from your pull request.

**Comment by barath** [comment]
You should also remove all the non-code files from your pull request.

File: src/service.cc
Comment (apetranik) [inline src/service.cc]: are these just to test or are you going to keep these?

File: src/service_controller.cc
Comment (apetranik) [inline src/service_controller.cc]: same here

File: src/service_controller.cc
Comment (apetranik) [inline src/service_controller.cc]: same here

**Review by apetranik — APPROVED** [review]
Looks good! Next commit I would remove the binaries

**Comment by cindyciclta** [comment]
I accidentally kept my debugging statements. I reminded myself to remove them before I pushed but I forgot about it. I pushed the updated ones now~

**Comment by cindyciclta** [comment]
I realized I wasn't supposed to merge until I got both approvals. I'm sorry.

**Pull request merged**
